server/raft.go

```diff
@@ -2430,7 +2501,7 @@ func (n *raft) runAsCandidate() {
 				n.trackPeer(vresp.peer)
 				votes++
 				if n.wonElection(votes) {
-					if votes == n.numActivePeers() || lxfer {
+					if true || votes == n.numActivePeers() || lxfer {
```

Comment:
That `true` was probably for a debug/test you were making...

Reply:
Ah yes. Will fix.